docker/Makefile

```diff
@@ -64,11 +68,22 @@ generate-production-static-data:
 	$(dcp) exec app python manage.py excel
 	$(dcp) exec app python manage.py price
 
+# drone should do this in working flow.
+# But if drone don't do this for some reasons,
+# you can run this recipe from local machine.
+prepare-deploy:
+	rm -rf front/build
+	$(dc) stop app && $(dcp) stop app
+	$(dc) rm -f app && $(dcp) rm -f app
```

Comment:
1. Why do you remove an `app` container?
2. You may use here `docker-compose rm -fs ...` to stop and remove container